app/View/Pages/help.ctp

```diff
@@ -107,10 +67,16 @@ $this->set('title_for_layout', $this->Pages->formatTitle(__('Help')));
                 __(
                     'If you have no idea what to do now that you are registered, '.
                     'you can introduce yourself on the <a href="{}">Wall</a>, or '.
-                    'join us on <strong>IRC</strong>. We will give you a purpose. :)',
+                    'join our <a href="{chatroom}">chatroom</strong>. We will '.
+                    'give you a purpose. :)',
                     true
                 ),
-                $this->Html->url(array('controller' => 'wall'))
+                $this->Html->url(
+                    array(
+                        'controller' => 'wall',
+                        'chatroom' => 'https://chat.tatoeba.org'
+                    )
+                )

```

Comment:
You need to create an array, since there are 2 variables in the string.
```
array(
    'wall' => $this->Html->url(array('controller' => 'wall')),
    'chatroom' => 'https://chat.tatoeba.org'
)
```